src/extension.ts

```diff
@@ -21,10 +22,13 @@ export function activate(context: ExtensionContext) {
             sonicPi.runCode();
         }, 500);
     });
-
+    let initOSC = commands.registerCommand('extension.initOSC', async () => {
+        await sonicPi.initOSC();
```

Comment:
I concerned about sonic pi's port change by it's reboot,
But I misunderstood vscode's activation event's running timing.
Since extension will be activated whenever a command is being invoked,
( https://code.visualstudio.com/api/references/activation-events#onCommand )
so registering initOSC is unnecessary as you pointed out.
I deleted those codes.